src/DirectWebServices/InvestmentsConnector/InvestmentsConnector.ts

```diff
@@ -75,20 +77,26 @@ export class InvestmentsConnector extends DWSConnector {
 
         super(options);
         this.options = options;
+        this.metadata = {
+            columns: {},
+            rawResponses: []
```

Comment:
No need for that. Take a look at the new metadata implementation in the lastest commit of the `feature/sectors-breakdown-converter` (https://github.com/highcharts/connectors-morningstar/pull/156/commits/d0a478306dfcce59af770464ad31ee5a60a29719).